### PR TITLE
Add dark mode

### PR DESCRIPTION
**Pull request by transitive-bullshit**
Check it out live here: https://test16.saas-journey.com 💯 

I added a commented-out example under `examples/pages/index.tsx` that shows how to enable `darkMode`.

Here's a test page for all the different Notion colors: https://test16.saas-journey.com/Color-Rainbow-54bf56611797480c951e5c1f96cb06f2

I'm using CSS variables to keep things DRY and so it'll be easier to add theming support in the future.

[Screen Shot 2020-07-22 at 3 09 40 AM](https://test16.saas-journey.com)

Screen Shot 2020-07-22 at 2 52 38 AM

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -1,8 +1,91 @@
+.notion {
```
Comment (transitive-bullshit) [inline src/styles.css:1]: I tried to keep the naming of the CSS variables consistent, but Notion's not super consistent internally.

**Comment by timolins** [comment]
Has a quick browse on my phone, looking good 😍 One thing I noticed is that the page emojis are translucent, probably due to the text color .

**Comment by transitive-bullshit** [comment]
@timolins good call -- just fixed so the icons & emoji will not be transparent in dark mode.

**Review by geekysrm — APPROVED** [review]

**Comment by transitive-bullshit** [comment]
For anyone ending up here, this functionality as well as a lot more improvements are available via https://github.com/NotionX/react-notion-x

**Comment by transitive-bullshit** [comment]
Hey @tobiaslins, would love to touch base with you guys at some point 😄

**Pull request closed without merging**